core/src/test/java/io/grpc/internal/InternalSubchannelTest.java

```diff
@@ -1145,6 +1146,8 @@ public void channelzMembership() throws Exception {
 
     MockClientTransportInfo t0 = transports.poll();
     assertTrue(channelz.containsClientSocket(t0.transport.getLogId()));
+    t0.listener.transportShutdown(Status.RESOURCE_EXHAUSTED);
+    fakeClock.forwardNanos(10);
```

Comment:
Why is advancing the time necessary?

Reply:
shutdown triggers connection backoff which won't be executed if not advancing the time. this fails the check in After block (checking any pending tasks)

Reply:
Go ahead and issue the ready, so the shutdown doesn't trigger a backoff. That is the more "normal" flow.

Reply:
okay. done.